shell/src/chain_manager.rs

```diff
@@ -610,19 +610,21 @@ impl ChainManager {
                 let block_header_with_hash = BlockHeaderWithHash::new(inject_data.block_header.clone()).unwrap();
                 let log = ctx.system.log().new(slog::o!("injection" => "block".to_string()));
 
+                // this should  allways return true, as we are injecting a forged new block
                 let is_new_block =
                     self.chain_state.process_block_header(&block_header_with_hash, log.clone())
-                        .and(self.operations_state.process_block_header(&block_header_with_hash))?;
+                        .and(self.operations_state.process_injected_block_header(&block_header_with_hash))?;
 
                 if is_new_block {
+                    info!(log, "NEW BLOCK DETECTED VIA INJECTION RPC!!!");
```

Comment:
@adonagy 
do you need this info?
do it as debug or trace pls. and normal camelCase :)

Reply:
Not really needed. Removed.